builder/build.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builder
```

Comment:
builder.Build stutters
Is there a better name?

Also missing package comment

Reply:
I agree it stutters, but I don't have a better name.  It feels like Reader.Read or Writer.Write.

Fixed missing package comment.

Reply:
either `getEvaluator` or `buildEvaluator` would be fine.